stack/stack.go

```diff
@@ -34,14 +35,20 @@ func getTemplateFromFile(context *context.Context) (string, string, error) {
 	return template, stackName, nil
 }
 
+// Looking for path to user/default template.
 func getPath(context *context.Context) (path string, err error) {
+	usr, userError := user.Current()
```

Comment:
We do have such facility in configuration, so please reuse that one instead of creating here a customized one.